theia-extensions/updater/src/electron-main/update/theia-updater-impl.ts

```diff
@@ -18,10 +18,12 @@ import { isOSX, isWindows } from '@theia/core';
 
 const STABLE_CHANNEL_WINDOWS = 'https://download.eclipse.org/theia/ide/version/windows';
 const STABLE_CHANNEL_MACOS = 'https://download.eclipse.org/theia/ide/latest/macos';
+const STABLE_CHANNEL_MACOS_ARM = 'https://download.eclipse.org/theia/ide/latest/macos-arm';
 const STABLE_CHANNEL_LINUX = 'https://download.eclipse.org/theia/ide/latest/linux';
 
 const PREVIEW_CHANNEL_WINDOWS = 'https://download.eclipse.org/theia/ide-preview/version/windows';
 const PREVIEW_CHANNEL_MACOS = 'https://download.eclipse.org/theia/ide-preview/latest/macos';
+const PREVIEW_CHANNEL_MACOS_ARM = 'https://download.eclipse.org/theia/ide-preview/latest/macos-arm/';
```

Comment:
It probably won't make a difference, but I noticed the other URLs don't have a trailing slash. Not sure if it needs to be adjusted in this case, just wanted to mention it.